databags/repos.json

```diff
@@ -13,7 +13,8 @@
             },
             "name": ".github",
             "url": "https://github.com/creativecommons/.github",
-            "website": ""
+            "website": "",
+            "slack": ""
```

Comment:
@sp35 you cannot add this here, this is an auto-generated file. you'll need to add slack channel info to individual repos in the `.cc-metadata.yml` file e.g. https://github.com/creativecommons/vocabulary/blob/master/.cc-metadata.yml and then update this script to read that info if it exists. https://github.com/creativecommons/ccos-scripts/blob/master/generate_projects_page/generate_projects_page.py

Reply:
Oh my bad, this should have been obvious, can't believe I did I this!